docker-compose.yml

```diff
@@ -1,36 +1,72 @@
 version: '2'
 services:
-  foodsoft:
+
+  #
+  # Foodsoft (main)
+  #
+
+  foodsoft_web:
```

Comment:
i'd prefer to name the "base" containers without `_web` suffix, since we also don't provide a `command: ./proc-start web`

Reply:
I'd rather be too clear. Even though it looks a bit ugly to reference `foodsoft_web` in `extend`, it actually is what we do.

Reply:
yes, but it's also the DEFAULT, so no need to name it explicit. if you want to be clear then you also need to at the `command`stuff, which is simply not necessary